contracts/src/periphery/ejection/IEigenDAEjectionManager.sol

```diff
@@ -0,0 +1,73 @@
+
+    /// @notice Sets the delay for ejection processes.
+    /// @param delay The number of seconds that must pass after initiation before an ejection can be completed.
+    ///              This is also the time guaranteed to a challenger to cancel the ejection.
+    function setDelay(uint64 delay) external;
+
+    /// @notice Sets the cooldown for ejection processes.
+    /// @param cooldown The number of seconds that must pass before a new ejection can be initiated after a previous one.
+    function setCooldown(uint64 cooldown) external;
+
+    /// @notice Starts the ejection process for an operator. Takes a deposit from the ejector.
+    /// @param operator The address of the operator to eject.
+    /// @param quorums The quorums associated with the ejection process.
+    function startEjection(address operator, bytes memory quorums) external;
+
+    /// @notice Cancels the ejection process initiated by a ejector.
+    /// @dev Any ejector can cancel an ejection process, but the deposit is returned to the ejector who initiated it.
+    function cancelEjectionByEjector(address operator) external;
+
+    /// @notice Completes the ejection process for an operator. Transfers the deposit back to the ejector.
+    /// @dev Any ejector can complete an ejection process, but the deposit is returned to the ejector who initiated it.
+    function completeEjection(address operator, bytes memory quorums) external;
+
+    /// @notice Cancels the ejection process for a given operator with their signature. Refunds the deposit to the recipient.
+    /// @param operator The address of the operator whose ejection is being cancelled.
+    /// @param apkG2 The G2 point of the operator's public key.
+    /// @param sigma The BLS signature of the operator.
+    /// @param recipient The address to which the gas refund will be sent.
+    function cancelEjectionWithSig(
+        address operator,
+        BN254.G2Point memory apkG2,
+        BN254.G1Point memory sigma,
+        address recipient
+    ) external;
+
+    /// @notice Cancels the ejection process for the message sender. Refunds gas to the caller.
+    function cancelEjection() external;
+
+    /// @notice Returns the address of the token used for ejection deposits and refunds.
+    function getDepositToken() external view returns (address);
+
+    /// @notice Returns the amount of deposit required for ejection processes.
+    function getDepositAmount() external view returns (uint256);
+
+    /// @notice Returns the address of the ejector for a given operator.
+    function getEjector(address operator) external view returns (address);
+
+    /// @notice Returns whether an ejection process has been initiated for a given operator.
+    function ejectionTime(address operator) external view returns (uint64);
+
+    /// @notice Returns the timestamp of the last ejection proceeding initiated for a given operator.
+    function lastEjectionInitiated(address operator) external view returns (uint64);
```

Comment:
Previously, there was a function `ejectionInitiated()` that returned a boolean value, depending on if there was currently an ejection in progress or not, but I don't see that function in this PR. Does `lastEjectionInitiated()` replace `ejectionInitiated()`? 

What is the behavior of this function if there was a previous ejection, but that ejection was canceled and is no longer active? Will it still return a timestamp?

Reply:
https://github.com/Layr-Labs/eigenda/pull/1639/commits/369158e6b0dd2648dc3c68dc1ad02993c8c7c1c6